_emails/kalamazooMi.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Kalamazoo, MI
+permalink: "/kalamazooMi"
+name: Letter to City and State Representatives
+state: MI
+city: Kalamazoo
+layout: email
+recipients:
+- mayor@kalamazoocity.org
+- knotte@kalamazoocity.org
+- urbanj@kalamazoocity.org
+- hessj@kalamazoocity.org
+- praedelc@kalamazoocity.org
+- cokcitymanager@kalamazoocity.org
+- Stephanie.Moore@kalcounty.com
+- Zachary.Bauer@Kalcounty.com
+- Tracy.Hall@kalcounty.com
+- Michael.Seals@kalcounty.com
+- Julie.Rogers@kalcounty.com
+- Jennifer.Aniano@kalcounty.com
+- Roger.Tuinier@kalcounty.com
+- John.Gisler@kalcounty.com
+- Christine.Morse@kalcounty.com
+- Mike.Quinn@kalcounty.com
+- Meredith.Place@kalcounty.com
+- SenSMcCann@senate.michigan.gov
+- jonhoadley@house.mi.gov
+subject: Regarding the funding of KDPS
+body: "Hello, \n\nI am a constituent living in Kalamazoo County and I am writing to you 
```

Comment:
make all of the fields here in brackets  all caps - e.g., [YOUR NAME]

Reply:
I'll assume you mean in general, these lines don't have any bracketed statements